README.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: >-
+விளக்கம்: >-
```

Comment:
`description:` is a gitbook identifier. It should be kept in English

Reply:
Wanted to understand, Where & How it will impact ??
Because [in the home page](https://tamil.coronasafe.network/) it was already updated ! & seems the placement is not broken.
@bodhish